spec/models/case_study_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe CaseStudy do
+  let(:content_item) do
+    GovukSchemas::Example.find("case_study", example_name: "doing-business-in-spain")
+  end
+
+  it_behaves_like "it has updates", "case_study", "has-updates"
+  it_behaves_like "it has updates", "case_study", "only-has-first-published-at"
+  it_behaves_like "it has no updates", "case_study", "doing-business-in-spain"
+  it_behaves_like "it has no updates", "case_study", "no-public-updated-at"
```

Comment:
The "no-public-updated-at" is bogus as there should always be a value for `public_updated_at` so I think this test and the associated example can be removed.